altair/utils/html.py

```diff
@@ -58,7 +58,7 @@
     });
     {%- endif %}
     require(['vega-embed'], function(vegaEmbed){
-    {%- endif %}
+    {%- endif %} (function(root) {
```

Comment:
Looks like the root variable is never used. What would you think about passing `vegaEmbed` so the same function definition can be used both with and without requirejs?

Also note that none of this javascript rendering code is tested as part of the automated tests, so it will be important to validate the outputs by exporting a chart with ``fullhtml`` and opening in a browser.

Reply:
@jakevdp  I have updated the PR.
* Change `root` to `vegaEmbed`
* Verify the exported html
![image](https://user-images.githubusercontent.com/164491/62177619-2c2b6680-b378-11e9-99e8-667d28bf3c9e.png)

Reply:
You'll also need to change the function argument to pass ``vegaEmbed`` rather than ``window``. And my suggestion was to combine it with the anonymous function definition just above to avoid code duplication.

Reply:
Thanks @jakevdp I have updated it

Reply:
Looks good.